Code/GraphMol/MolOps.cpp

```diff
@@ -998,17 +998,20 @@ void addHapticBond(RWMol &mol, unsigned int metalIdx,
   unsigned int numbonds = mol.addBond(dummyIdx, metalIdx, Bond::DATIVE);
   auto bond = mol.getBondWithIdx(numbonds - 1);
 
-  // Get the atom numbers for the end points.
+  // Get the atom numbers for the end points.  First number is the
```

Comment:
Sorry, this should have been caught in the review of the initial PR

Reply:
Yeah, but given that I'd just written my own parser for it, I might have been expected to get it right the first time!